iroha/src/wsv.rs

```diff
@@ -32,26 +25,34 @@ impl WorldStateView {
     }
 
     pub fn add_domain(&mut self, domain: Domain) {
-        self.domains.insert(domain.name.clone(), domain);
+        self.peer.domains.insert(domain.name.clone(), domain);
```

Comment:
Why not use `self.peer += domain`? You implemented `AddAssign` in `peer.rs`.

Reply:
Decided to remove this impl

Reply:
I'm not sure what adding a peer and a domain means actually. We should discuss it.